src/meta/model_v2/src/index.rs

```diff
@@ -27,7 +27,7 @@ pub struct Model {
     pub index_table_id: TableId,
     pub primary_table_id: TableId,
     pub index_items: ExprNodeArray,
-    pub original_columns: I32Array,
+    pub index_columns_len: u32,
```

Comment:
```suggestion
    pub index_columns_len: i32,
```
The column type from the sql-backend can only be derived as `i32`.